package.json

```diff
@@ -21,7 +21,11 @@
     "serve": "./entrypoint",
     "test-js": "vitest --run",
     "test-e2e-edge": "npx playwright test --project chromium:lxd-latest-edge firefox:lxd-latest-edge",
-    "test-e2e-stable": "npx playwright test --project chromium:lxd-5.0-stable firefox:lxd-5.0-stable"
+    "test-e2e-stable": "npx playwright test --project chromium:lxd-5.0-stable firefox:lxd-5.0-stable",
+    "test-js-coverage": "vitest --run --coverage",
+    "test-e2e-coverage": "yarn test-e2e-coverage-playwright ; yarn test-e2e-coverage-report",
+    "test-e2e-coverage-playwright": "rm -rf coverage/playwright* ;  PW_TEST_HTML_REPORT_OPEN='never' npx playwright test --project coverage",
+    "test-e2e-coverage-report": "nyc report --reporter html --reporter cobertura --reporter text --reporter text-summary --temp-dir coverage/playwright --report-dir coverage/playwright-report"
```

Comment:
why so many reporters? Is it for html from playwright and the rest is for vitest?

Reply:
The reporters are quite cheap if all are in one nyc run. As for why so many:
- html for us to look at on github pages
- xml for an external dashboard to import
- text and text-summary to have the output available directly on the github action